src/components/molecules/UserList/UserList.tsx

```diff
@@ -56,7 +56,7 @@ const UserList: React.FunctionComponent<PropsType> = ({
   if (!usersSanitized || attendance < 1) return <></>;
   return (
     <>
-      <div className="userlist-container">
+      <div className="container userlist-container">
```

Comment:
This is Bootstrap's `.container` class. We use `.row` and `.column` below. Since these have negative margins, things look weird when the `.container` class isn't used.